### PR TITLE
Refactor delete

### PR DESCRIPTION
**Pull request by frankyma**
Following the steps defined in the ticket description.  To use `deleteNodeNew` will need to pass in all the baseNode labels, and a dictionary of all of its unique property relations and those property node labels.

### CODE REVIEW

**Review by michaelmarshall — CHANGES_REQUESTED** [review]
Looks good so far. Now that we've gotten here, please add `DeleteX` members to the `Powers` enum. Then, in the delete function, check if a user has that power by calling the authorization service function. For now you may want to grant all users the power you are creating by going to the user service create function and adding the power to the list that all users will get. Once that is done, unskip the delete test for each object you have covered. Please add this list to your ticket too. Thanks Frank!

**Comment by michaelmarshall** [comment]
By `DeleteX` I mean `DeleteLanguage`, `DeleteUser`, etc. Whatever the base node is.

**Comment by frankyma** [comment]
Pushed the changes to add the check `Powers`.  I only added the `DeleteLanguage` power to keep this PR small, once this gets merged we can do the rest in a batch

**Review by michaelmarshall — APPROVED** [review]

File: .eslintrc.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ rules:
       filter: __typename
       format: ~
     - selector: property
-      format: [camelCase, PascalCase]
+      format: [camelCase, PascalCase, snake_case]
```
Comment (CarsonF) [inline .eslintrc.yml:26]: Why was this needed?

Comment (frankyma) [reply .eslintrc.yml:26]: This showed up as a warning: https://github.com/SeedCompany/cord-api-v3/pull/1320/files#diff-3db627b34674b6a447753880e259bffb58dfe6cd8f22f35623415bce6a93b455R911

And I though if snake case will be in our database from now on might as well add the rule.
